docs/plugins/fluentd/common/buffer.md

```diff
@@ -28,7 +28,7 @@ Buffer defines various parameters for the buffer Plugin
 | flushAtShutdown | Flush parameters This specifies whether to flush/write all buffer chunks on shutdown or not. | *bool |
 | flushMode | FlushMode defines the flush mode: lazy: flushes/writes chunks once per timekey interval: flushes/writes chunks per specified time via flush_interval immediate: flushes/writes chunks immediately after events are appended into chunks default: equals to lazy if time is specified as chunk key, interval otherwise | *string |
 | flushInterval | FlushInterval defines the flush interval | *string |
-| flushThreadCount | The number of threads to flush/write chunks in parallel | *integer |
+| flushThreadCount | The sleep interval (seconds) for threads to wait for the next flush try(when no chunks are waiting) | *string |
```

Comment:
There seems to be some pr being rubbed together here. For example, this : https://github.com/fluent/fluent-operator/pull/527

Reply:
I think that's because the CRD docs are supposed to be auto-generated from the inline comments of the go object, so the person should've changed the comment instead of directly changing the file here. I can accommodate that change in this PR.

Reply:
@wenchajun fixed in subsequent commit

Reply:
@wenchajun @benjaminhuo please review further, I'll be happy to incorporate any suggestions and iterate on the review.